test/Rx/Functional/FunctionalTestCase.php

```diff
@@ -99,4 +112,72 @@ protected function createTestScheduler()
     {
         return new TestScheduler();
     }
+
+    protected function convertMarblesToMessages(string $marbles, array $eventMap = [], \Exception $customError = null, $subscribePoint = 0)
+    {
+        var_dump($eventMap);
+        /** @var Recorded $events */
+        $events = [];
+        $zero = 0;
+
+        for ($i = 0; $i < strlen($marbles); $i++) {
+            switch ($marbles[$i]) {
+                case '-': // nothing
```

Comment:
In RxJS 5 they very often use also spaces " " (https://github.com/ReactiveX/rxjs/blob/master/spec/operators/window-spec.ts).

This is what I used in #144 

```
case '-': // nothing
case ' ':
    continue;
```

https://github.com/martinsik/RxPHP/blob/002671800236c56d872a68dc635a5836c4f39448/test/Rx/Functional/FunctionalTestCase.php#L156